docs/services/valkey-connect.md

```diff
@@ -0,0 +1,52 @@
+# Connecting to a Valkey Server
+
+This document outlines how to connect to a Valkey server using two methods: 
+* `valkey-cli` command-line interface
+* Python with `redis-py` library
+
+## 0. Get the endpoint
+
+To get the endpoint to connect to, go to [Ivee portal](https://app.ivee.cloud) and in the "Instances" choose the instance. 
+In the "Connectivity" section you will find the Global endpoint and the Port.
+
+## 1. Connect with `valkey-cli`
+
+`valkey-cli` is a command-line interface for interacting with Valkey servers. I
+t allows you to execute commands, inspect data, and manage your Valkey instance.
```

Comment:
command-line interface (cli). 
The cli lets you execute commands, and so on

Reply:
Thanks! Fixed.